packages/radix-ui-themes/src/helpers/breakpoints.ts

```diff
@@ -1,5 +1,9 @@
+import { hasOwnProperty } from './has-own-property';
+
 type Breakpoints = 'initial' | 'xs' | 'sm' | 'md' | 'lg' | 'xl';
 type Responsive<T> = T | Partial<Record<Breakpoints, T>>;
+type Wide<T extends string> = T | Omit<string, T>;
+type ResponsiveWide<T extends string> = Responsive<Wide<T>>;
```

Comment:
Do these names make sense? Any better ideas @benoitgrelard?

Reply:
Not really no, what are we trying to convey here?
- an enum with a list of options
- but also the rest of all strings
- does it have to be responsive?

Reply:
> - an enum with a list of options
> - but also the rest of all strings

Yep!

> does it have to be responsive?

Doe not **have** to be, it theory we could have a prop that’s just `Wide` but not `ResponsiveWide`. However, I'm not seeing a case for one yet. All the props that will support wider values happen to be responsive.